CMakeLists.txt

```diff
@@ -164,6 +164,9 @@ ENDIF()
 IF((CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX) AND NOT CMAKE_COMPILER_IS_MINGW)
   IF (BUILD_SHARED_LIBS AND CMAKE_SIZEOF_VOID_P EQUAL 8) # -fPIC is only required for shared libs on 64 bit
      SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+     IF(ASSIMP_BUILD_ZLIB)
+	     SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
```

Comment:
That might well be. I'm just a newb to C/C++ development, this was the minimum circumstance that would allow compilation in the configuration I was testing. 

When doing the searching to figure things out this far, I found that -fPIC will inflate the size of the *.o files. I doubt that the few C files affected would be significant when all of the C++ are already using the option. Don't know enough to evaluate any other possible side effects.

BTW, I think this would have been caught automatically if the travis build had a configuration with `ASSIMP_BUILD_ZLIB=ON` would that be worth adding? I can run up a separate PR if you like.